coffee-shop-kata/SETUP.md

```diff
@@ -1,22 +1,22 @@
 ## Setting up the project
 
 ### Requirements
-* [JDK 20](https://jdk.java.net/20/) installed on your computer 
+* [JDK 21](https://jdk.java.net/21/) installed on your computer 
   
   Note: The latest version of IntelliJ only supports Java 20
 * [Maven 3.6.1+](https://maven.apache.org/download.cgi) installed on your computer
-* [IntelliJ IDEA 2023.1+](https://www.jetbrains.com/idea/download/?section=windows) or similar IDE for Java
+* [IntelliJ IDEA 2023.2.5+](https://www.jetbrains.com/idea/download/?section=windows) or similar IDE for Java
```

Comment:
can you remove the note that says the latest version of intellij only supports Java 20?

Reply:
Yes, thanks for catching that!

Reply:
Fixed